presto-main/src/main/java/com/facebook/presto/server/security/LdapServerConfig.java

```diff
@@ -20,10 +20,10 @@
     private String ldapUrl;
     private String baseDistinguishedName;
     private boolean authenticationEnabled;
-    private String serverType = ServerType.ACTIVE_DIRECTORY.name();
+    private String serverType;
```

Comment:
I would still keep these default values, so that the user doesnt have to set these for AD